eth/transaction.go

```diff
@@ -28,6 +33,13 @@ type Transaction struct {
 	R           Quantity  `json:"r"`
 	S           Quantity  `json:"s"`
 
+	// Gas Price (optional since not included in EIP-1559)
+	GasPrice *Quantity `json:"gasPrice,omitempty"`
```

Comment:
making sure I understand this correctly for TLM, but it looks like GasPrice is "going away" and being replaced with MaxFeePerGas & MaxPriorityFeePerGas for EIP-1559. Is there a case we would get all 3 values in a txs?

Reply:
Nope, it should be either GasPrice alone for legacy txs or `MaxFeePerGas` and `MaxPriorityFeePerGas` for -1559 txs.